flipperdevices/flipperzero-good-faps
# [FL-3667] NFC Magic fix gen4 detection

# What's new

- Add gen4 password for gen4 scanner

# Verification 

- Read gen4 tag.
- Set new password for gen4 tag.
- Enter password manually in start scene.
- Verify gen4 card is detected

# Checklist (For Reviewer)
